executor/oci/spec.go

```diff
@@ -196,7 +197,7 @@ func GenerateSpec(ctx context.Context, meta executor.Meta, mounts []executor.Mou
 		}
 	}
 
-	if tracingSocket != "" {
+	if tracingSocket != "" && runtime.GOOS != "freebsd" {
```

Comment:
It looks like this could maybe use `nullfs` as well? Or is there some other tracing limitation for freebsd, in which case we should add an explicit comment.

Reply:
We should avoid the explicit check here anyways, we should change the signature of `getTracingSocketMount` to return a pointer, so we can return nil and skip adding the mount in this case (since we have the compile time inclusion).